src/app/topics/topics.component.html

```diff
@@ -16,6 +17,13 @@
 					<span (click)="like('like', topic._id)" class="topic__likes-up">&uarr;</span>
 					<span (click)="like('dislike', topic._id)" class="topic__likes-down">&darr;</span>
 				</td>
+        <td *ngIf="auth.authenticated">
```

Comment:
Let's probably just disable the date input, not hide it at all, if user is not authenticated.